### ISSUE TITLE
Can't install the required modules

### ISSUE THREAD

**Issue by usr577** [neonbjb/tortoise-tts#449]
So I can't install the scipy 0.10.1 on python3.9 so I gogled and it said that it works with versions up to python3.4 so I tried that but I can't get python3.4 to install pip because of some error with ssl and I have a question: What python version is this code designed to run on?

**Comment by neonbjb** [comment]
I've run it on 3.6 and 3.9, I'm quite sure it works on newer versions too.

**Comment by usr577** [comment]
If I install scipy 0.10.1 it returns `ModuleNotFoundError: No module named 'numpy'` even though I did `pip3.9 install numpy==1.20.0` (1.20.0 is the version in the "requirements.txt")  beforehand

**Comment by tiebingzhang** [comment]
I doubt the author means version 0.10.1 scipy since it's released 11 years ago. I changed the version number to ` 1.10.1` in the `requirements.txt` and it's working fine for me.

**Comment by usr577** [comment]
Ok Thanks, I knew it was old but I tried to install it anyways because it was in the requirements.txt

**Comment by usr577** [comment]
Working now but Ubuntu keeps killing the process, because I ran out of Memory, what amount of memory does this use?

**Comment by yatesdr** [comment]
It uses about 6gb of vram for me making some short sentences, but it can vary depending on the samples and text length you're using.

**Comment by usr577** [comment]
No, I meant normal ram, not vram, sorry if I did't make clear.

**Comment by yatesdr** [comment]
I haven’t formally monitored for that, but it doesn’t seem to use a lot of regular ram.   It may spike when it’s loading the model to vram, but I’d be really surprised if it used more much more ram than vram for this.   As a rule of thumb you probably want at least 32gb or twice as much regular ram as you have vram, but more won’t hurt.   I ran on 64gb for a while until I got a good deal on another 64, but it’s rare that it ever uses much more than about 25gb unless I’m doing something unusual.
